test/apicoverage-webhook/kodata/LICENSE

```diff
@@ -0,0 +1 @@
+../../../LICENSE
\ No newline at end of file
```

Comment:
<!--whitespace-bot-->
```suggestion
../../../LICENSE

```

Reply:
I need to figure out how to filter these.